packages/core-web/dist/js/markbind.min.js.LICENSE.txt

```diff
@@ -8,28 +8,3 @@
   * https://github.com/linusborg/portal-vue
   * 
  */
-
```

Comment:
Please refer to https://markbind-master.netlify.app/devguide/workflow#editing-frontend-features
for the details on how frontend features are updated. I think this file change and other ones in `dist` should not be included in this PR.